src/content/docs/en/reference/experimental-flags/responsive-images.mdx

```diff
@@ -33,22 +41,37 @@ Additionally, Astro's image components can receive [responsive image props](#res
 
 Images in your `public/` folder are never optimized, and responsive images are not supported.
 
-## Responsive image configuration settings
+:::note
+Enabling responsive images will generate additional image sizes for all affected images. For prerendered pages this happens during the build so may increase the build time of your project, especially if you have a large number of images. For pages rendered on-demand the images are generated as-needed, so this has no impact on build times but may increase the number of transformations performed. Depending on your image service this may incur additional costs. 
+:::
+
+## Image layout
+
+In order to generate the correct `srcset` and `sizes` attributes, the `<Image />` and `<Picture />` components need to know how the image should resize when its container changes size. This is done by setting the `layout` prop, or `image.experimentalLayout` default. The supported values are:
+
+- `constrained` - The image will scale to fit the container, maintaining its aspect ratio, but will not exceed the specified `width` and `height`, or the image's original dimensions.
```

Comment:
```suggestion
- `constrained` - The image will scale to fit the container, maintaining its aspect ratio, but will not exceed the specified `width` and `height`, or the image's original dimensions. Use this for images that should be responsive, but only when smaller than the original (or a specified) size.
```
I like how the others had a "Use this/this is useful" so something like this maybe?

Reply:
I've done new wording, including saying that if they're unsure they should choose this, and that it matches the default Tailwind styles